src/app/public/modules/tabs/tabset.component.ts

```diff
@@ -50,7 +63,7 @@ import {
   ]
 })
 export class SkyTabsetComponent
```

Comment:
Are we concerned at all that this is technically going to break any bookmarks that were made using the previous version of this functionality? I don't think it will stop any links from working but will cause them not to go to the tab that was bookmarked. Thoughts?

Reply:
Another good observation. I'm not sure of a way around it, unfortunately, since the tabset shouldn't be affecting the "real" route to begin with. I'm going to guess that very few people (if any) are using this functionality because as it stands today it's rather broken.

Reply:
Sounds good to me